pandas/tests/reshape/concat/test_concat.py

```diff
@@ -698,3 +698,13 @@ def test_concat_posargs_deprecation():
         result = concat([df, df2], 0)
     expected = DataFrame([[1, 2, 3], [4, 5, 6]], index=["a", "b"])
     tm.assert_frame_equal(result, expected)
+
+
+def test_concat_series_with_decreasing_rangeindex():
+    # GH#41965
+    ser1 = Series([], index=pd.RangeIndex(0), dtype="int64")
```

Comment:
can you test the reverse a postivive step index as well)

Reply:
added